src/format/strings.json

```diff
@@ -13,6 +13,19 @@
             "receipt": "🧾",
             "grey_question": "❔"
         },
+        "accessible": {
+            "x": "❌",
+            "arrow_up_small": "🔺",
+            "small_red_triangle_down": "🔻",
+            "green_circle": "✅",
+            "yellow_circle": "⚠️",
+            "red_circle": "❎",
+            "hatching_chick": "🐣",
```

Comment:
This emoji (🐣) is quite verbose when read: "newly hatched chick still in shell."

What do we think about 🆕, instead?